src/Bundle/CoreBundle/Field/Types/LinkFieldType.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: stefankamsker
+ * Date: 17.09.18
+ * Time: 13:01
+ */
+
+namespace UniteCMS\CoreBundle\Field\Types;
+
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+use UniteCMS\CoreBundle\Field\FieldableFieldSettings;
+use UniteCMS\CoreBundle\Entity\FieldableField;
+use UniteCMS\CoreBundle\Form\LinkType;
+use UniteCMS\CoreBundle\Field\FieldType;
+
+class LinkFieldType extends FieldType
+{
+    const TYPE = "link";
+    const FORM_TYPE = LinkType::class;
+
+    /**
+     * All settings of this field type by key with optional default value.
+     */
+    const SETTINGS = ['allow_title', 'allow_target'];
```

Comment:
I don't think, that "allow" is a good, I would go with "title_input", "target_input" or "title_widget", "target_widget". What do you think?